app/domain/operations/hbx_admin/dry_run/individual/analyzer.rb

```diff
@@ -0,0 +1,195 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+
+
+module Operations
+  module HbxAdmin
+    module DryRun
+      module Individual
+        # This Operation is responsible for getting the renewal dry run count for applications, notices and benefits.
+        class Analyzer
+          include Dry::Monads[:result, :do]
+
+          APPLICATION_STATES = %w[
```

Comment:
@vkghub Instead of hard coding the aasm states. We can define a class method in the Application model like the one below and call the method everywhere we need the aasm_states. We do not have to update/maintain this constant as we introduce new states. It is okay to add the `draft` state to this list.

```ruby
def self.all_aasm_states
  aasm.states.map(&:name)
end
```

Reply:
I like this method.